crates/turborepo-analytics/src/lib.rs

```diff
@@ -1,5 +1,10 @@
 #![deny(clippy::all)]
 
+//! Turborepo's analytics library. Handles sending analytics events to the
+//! Vercel API in the background. Currently we only record cache usage events,
+//! so when the cache is hit or missed for the file system or the HTTP cache.
+//! Requires the user to be logged in to Vercel.
```

Comment:
```suggestion
//! HTTP cache usage requires the user to be logged in to Vercel.
```

Reply:
We actually require the user to be logged in for any analytics to be sent

Reply:
ah ok, we're recording local cache usage stats also?